src/libraries/System.Private.DataContractSerialization/src/System/Xml/XmlBaseReader.cs

```diff
@@ -2808,10 +2808,7 @@ public void Clear()
             {
                 if (_nsCount != 0)
                 {
-                    for (int i = 0; i < _shortPrefixUri.Length; i++)
-                    {
-                        _shortPrefixUri[i] = null;
-                    }
+                    Array.Clear(_shortPrefixUri, 0, _shortPrefixUri.Length);
```

Comment:
Should we open an issue to expose Array.Clear(Array) publicly?

Reply:
Pulled out into https://github.com/dotnet/runtime/issues/51581.